Curvature/HTTPRequest/HTTPRequest.swift

```diff
@@ -32,6 +32,7 @@ public struct HTTPRequest {
 
     public var parameters: [String: String] = [:]
     public var data: [String: Any] = [:]
+    public var context: [String: Any] = [:]
```

Comment:
I guess this is a bigger discussion on how to streamline deserialisation when building an API that consumes both XML and JSON. I'll remove the data property for now.